wifiphisher/pywifiphisher.py

```diff
@@ -297,6 +298,17 @@ def try_change_mac(self, iface_name, mac_address=None):
             print("[{0}!{1}] {2}").format(R, W, err)
 
     def start(self):
+        dependency_result = is_all_dependencies_installed()
+        if not dependency_result.status:
+            logger.error("The following dependencies are missing:")
+            print("The following dependencies are missing:")
```

Comment:
If the message that you are planning to log is the same with the message that you want to print, it makes more sense to construct the message, store it in a variable and then print/log that variable.